content/scout/exceptions.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Manage vulnerability exceptions
+description: |
+  Exceptions let you provide additional context and documentation for how
+  vulnerabilities affect your artifacts, and provides the ability to
+  suppress non-applicable vulnerabilities
+keywords: scout, cves, suppress, vex, exceptions
+---
+
+Vulnerabilities found in container images sometimes need additional context.
+It's often the case that vulnerabilities require more nuance than a CVSS score
+or a severity rating, when viewed in the context of an software application.
+
+**Exceptions** provide a way to describe this additional context, and attach
```

Comment:
I'm not sure this is quite right.

- Exceptions are specifically about allowing an organisation to document reasons why a particular vulnerability instance does not need to be addressed. It doesn't allow for attaching purely informational context.
- Exceptions are also not something we can really attach to images, they are a set of Scout-specific rules for how Scout should treat certain vulnerabilities in certain images.

This might be down to some confusion about the difference between VEX statements and Exceptions?

Reply:
While M1 only allows for Exceptions based off VEX statements, conceptually they are two different things.

Reply:
thanks @chrispatrick - sorry for convoluting stuff. I've removed the bits talking about "acceptable risk" and focused strictly on negating false positives.

For the time being, I'd like to clearly call out that we achieve this using VEX documents. When that changes and we have other ways of managing exceptions, we'll get this doc updated to reflect the different options available.